index.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width= , initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css" integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="./style/main.css">
+    <title>KMarketer</title>
+</head>
+<body>
+    <div class="content">
+        <div class="header header--bg-gray">
+            <div class="header__item ">
+                <div class="logo">
+                    <h1 class="">KMarketer</h1>
+                </div>
+                <ul class="navigation">
+                    <li class="navigation__item">
+                        <a class="link link--disabled" href="#">Home</a>
```

Comment:
`link` is good , however i can suggest you to reference it to its block like `navigation__link` then the modifier will be `navigation__link--disabled`